packages/spectral-config/package.json

```diff
@@ -2,12 +2,16 @@
   "name": "@readme/spectral-config",
   "version": "3.0.6",
   "description": "ReadMe coding standards for API documentation",
-  "main": "src/index.js",
+  "main": "dist/index.cjs",
+  "module": "./dist/index.js",
+  "type": "module",
   "engines": {
     "node": ">=16"
   },
   "scripts": {
+    "build": "tsup",
     "lint": "eslint .",
+    "prepack": "npm run build",
```

Comment:
@erunion is this the only thing needed for lerna to build the dist before publishing?

Reply:
might add a `prebuild` for blowing away the dist directory too. i dont know if the tsup `clean` directory will do that or not.

```
"prebuild": "rm -rf dist/"
```

Reply:
i think we should be okay on that front b/c `clean: true` in the `tsup` config